spec/codebraker/lib/codebraker_console_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+RSpec.describe CodebrakerConsole do
+  describe '#exit' do
+    let(:console) { described_class.new }
```

Comment:
use `subject`
https://stackoverflow.com/questions/38437162/whats-the-difference-between-rspecs-subject-and-let-when-should-they-be-used#:~:text=Summary%3A%20RSpec's%20subject%20is%20a,understand%20and%20should%20be%20avoided.